app/src/main/java/com/iron/espresso/presentation/sign/SignUpViewModel.kt

```diff
@@ -0,0 +1,77 @@
+package com.iron.espresso.presentation.sign
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class SignUpViewModel : ViewModel() {
+
+    val signUpEmail = MutableLiveData<String>()
+    val signUpNickname = MutableLiveData<String>()
+    val signUpPassword = MutableLiveData<String>()
+    val isProgressVisible = MutableLiveData<Boolean>()
+
+    private val _checkType = MutableLiveData<CheckType>()
+    val checkType
+        get() = _checkType
+
+    private val _exitIdentifier = MutableLiveData<Boolean>()
+    val exitIdentifier
+        get() = _exitIdentifier
+
+
+    val checkEmail: Function1<String, Unit> = this::verifyEmailCheck
```

Comment:
```suggestion
    val checkEmail: (email: String) -> Unit = this::verifyEmailCheck
```
요런애들
이런 람다형태가 좀 더 코틀린 스럽다 싶긴해서 남겨봐요

Reply:
54762e2 적용하였습니다.

Reply:
5c1927c 모두 수정하였습니다.